server/src/internalClusterTest/java/org/opensearch/search/fetch/subphase/highlight/HighlighterSearchIT.java

```diff
@@ -2282,6 +2282,7 @@ public void testHighlightNoMatchSizeNumberOfFragments() throws IOException {
         assertHighlight(response, 0, "text", 1, 2, equalTo("This is the <em>fifth</em> sentence"));
     }
 
+    @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/issues/2063")
```

Comment:
Just a question: why not to fix the tests in the same pull request vs doing cleanup after? (personally, I would prefer to see tests passing, not muted).

Reply:
I'm looking at some of the failing tests while the PR is open but I went ahead and created that punch list since this PR is massive as is and I like the idea of sharing the work :)

Reply:
May be could help you by contributing to this pull request directly?

Reply:
:+1: I am in, will take the JDK-11 build issues and HighlighterSearchIT